### ISSUE TITLE
Thank You Feodor2

### ISSUE THREAD

**Issue by E-System33** [Feodor2/Mypal#4]
Thank you for making this browser that brings XP/Vista back to life. 

I'm really sorry about the last event.

But I think the new Mypal will be better.

Don't lose courage we are here to encourage you ;)

Good continuation

A french Mypal user

**Comment by Feodor2** [comment]
If someone still want to continue updating of Mypal let me know. No coding skill is required but desirable.
But you are to be very careful, sure you become their next target.

**Comment by Leifman27** [comment]
@Feodor2 I have 2 questions : 

1. All they want is for u to post the 'source' with every version and for ever version available. is there a reason you don't want to do it? because from what they said.. if you do that... they will never have any problem with you.

2. Different to what i wrote above in '1' , is there a way to contact you and simply maybe even pay for u to make new builds and get them privately? because i really appreciate 'mypal' and even if you can't post it publicly or on github... i'd still be interested in having access and following your work!

**Comment by E-System33** [comment]
> 
> 
> Si quelqu'un souhaite continuer à mettre à jour Mypal, faites-le moi savoir. Aucune compétence en codage n'est requise mais souhaitable.
> Mais vous devez être très prudent, vous deviendrez certainement leur prochaine cible.

Not me because I do not know how to code.
But there are very good people who will do it well ;)

**Comment by chermany4ever** [comment]
Thanks again Feodor! I second all motions here. We need mypal to continue.

**Comment by Feodor2** [comment]
For just plain update from palemoon source coding skills not required at all

>All they want is for u to post the 'source' with every version and for ever version available.

I think they will seek any opportunity to destroy any winxp browser based on their code, you do good and post source code right,  but might make the mistake in other things, so to be very careful.

**Comment by chermany4ever** [comment]
I have always thought that Github was da shit. Was I wrong.

**Comment by E-System33** [comment]
It's mostly because of Palemoon

**Comment by chermany4ever** [comment]
Those Palemoon idiots. Nobody is using that fkn' sht anymore. Well... maybe if you use Puppy Linux.

**Comment by NEO-SPECTRUMAN** [comment]
>For just plain update from palemoon source coding skills not required at all
для собирания среды для сборки левых проджектов
нужны скилы по более чем уменее кодить
я то кодить вроде немного уметь
но большую часть мракобесия собрать не могу
изза тон левых зависимостей...

**Comment by NEO-SPECTRUMAN** [comment]
а так это будет сильный удар по ХР
последний их норальных firefox-ов добит...

**Comment by greyfox17** [comment]
I am really sorry that this happened although I am not sure what exactly happened. In any case I would like to thank you for this browser. It is amazing. I just created a GitHub account to express my gratitude.

**Comment by NEO-SPECTRUMAN** [comment]
теперь все надежды перекладываются на полумертвый one-core api
сборки ncore которых давно нет
и всякие там shorthorn-ы...

**Comment by yongsu101** [comment]
thanks for everything from me too, feodor my friend
sorry i talked too much in the license issue but i was angry with their bs and lies
i hope you can get a better version of mypal for xp now instead of their crap
but if someone keeps old mypal alive that would be great too
to have matt tobin squirming is always a good thing
maybe good idea to copy over his first attack too to keep full record
i mean issue #35
so everyone can see how well you behave when the request isn't total bs
you're a great guy and deserve a great life and happiness
if i win a million bucks i will donate to you a good sum

**Comment by ghost** [comment]
Don't let the pale moon virgins push you around. They're bluffing, there's no way they will (or are capable of) pressing charges.

**Comment by dca00** [comment]
Enough had been said about Palemoon cabal's scare campaign against forks, but here is my 2c:

Agressive, "lawyer-like" and "cop-like" comments and abusive language of the trio has no merit whatsoever. Their main claim was that source code was not made immediately available to anyone who d/l the binary. It is without merit since all source code has always been avalable. The claim that it was only available in its entirety upon request is without merit since MPL is clear:

3.2. Distribution of Executable Form
If You distribute Covered Software in Executable Form then:
    such Covered Software must also be made available in Source Code Form

It means exactly what it says: "must also be made available". It does not say how. It only says that the source code must be accessible to anyone and everyone interested in it. That had been followed by Feodor2. Therefore, the whole assault on him is totally without merit.

There are plenty of examples, over the course of the history of open-source when source code was not immediately available to the recipient of the binary format. There are plenty of examples where source code was not readily compilable by anyone but the developer. None of them had gotten prosecuted successfully for just that because it is impossible. Even Linux kernels of many of the distros cannot be built outside of the maintainer's own lab. No one goes around prosecuting them for just that.

There are plenty of Linux-based pieces of hardware for which no source code is provided. They make millions and billions of US$ but no one prosecutes them for just not providing source code.

Bottom line is, Palemoon cabal's scare campaign is not defensible in court. Do not bend for it.

**Comment by ByRodrigoSoares** [comment]
It would be possible @Feodor2 to create a forum and continue updates for Mypal fans and provide the download links by email only so Palemoon ordinarys wouldn't interfere

**Comment by rubyFeedback** [comment]
The reason I upvoted dca00 is not because of all of the content, e. g. the claim about the court I consider incorrect.

However had I agree on the general notion due to this comment:

https://github.com/Feodor2/Mypal/issues/5

In quote: "I'm telling anyone reading this so there's no doubt about the current state of it and so people are informed who might skim over those details and in turn become in violation themselves. Nobody wants that."

This is indeed FUD, the part "in turn become in violation".

**Comment by BokeeXD** [comment]
> Those Palemoon idiots. Nobody is using that fkn' sht anymore. Well... maybe if you use Puppy Linux.

well palemoon itself does suck, but forks that are made for older operating systems are decent

**Comment by BokeeXD** [comment]
this is fucking terrible, palemoon devs are like discord mods really, hope for the best in the future

**Comment by ghost** [comment]
Feodor, я поддерживаю тебя, палемуновцы те еще бяки, но справедливости ради, до всей этой петрушки, некоторые люди действительно ждали от тебя ссылку на архив с исходным кодом, патчами и подробную инструкцию по компиляции, с расказом обо всех тонкостях компилирования столько сложного софта в столь непростых условиях. Только бинарь это тоже хорошо, но... Например, я слышал такое от печально известного пох'а с опеннета. Просто есть люди, которые любят возиться, компилировать сами. Тем более под xp это увлекательное занятие. Палемуновцы бы все равно придрались, конечно, но речь не о них.

**Comment by Sen-nin** [comment]
Thank you Feodor2 for developing MyPal. I would really like to continue your project, and I would like to distribute MyPal under the laws of my country. I may be their target, but I always have several lawyers working for me anyway. You can contact me via my twitter account.

**Comment by SaranSDS008** [comment]
> this is fucking terrible, palemoon devs are like discord mods really, hope for the best in the future

Yeah, Mattobin and his goons are such a Fking Immature Idiots just like Discord and Reddit Mods. Just fking sickens me.

**Comment by ghost** [comment]
There is no conceivable way, in my opinion, that these "people" (fat ******* manchildren) can file a civil lawsuit (there is no "pressing charges" because it is not a criminal cause), let alone in Russia of all places. Maybe they could try to file a suit in the states, but I doubt the original prayer would even be coherent. You should not allow fat losers to bully you on the internet. They do not have enough money, I would think, to do something like this. It would cost at least 10,000 in the USA, much more internationally. This is not legal advice, just my two cents. In my opinion, they are the ones who have violated the MPL by attempting to misuse it in bad faith to engage in monopolistic practices and to use Mozilla source code in a quasi-proprietary fashion. 

This problem is more than just myPal. People need to be able to use old operating systems. It is a right to repair issue. If you don't try to fight this IMO you are a bit unethical. You should reach out to Louis Rossman / RMS etc. 

Do you need money for a lawyer? I would be willing to donate to you and I am sure many people would also

You don't need to wait for them -- you could, in my opinion, consider bringing a civil suit against them in the USA. It is comparatively easier to find where to serve _**them**_ process (rather than where to serve you). In essence, people who are using mypal probably have tangible and non-tangible monetary / psychological losses from the project being forcibly shutdown. Any person who has these losses, I think, could be named as a plaintiff. I'd say that you could name me as a plaintiff, but I do not know if I use mypal enough to have significant losses. 

PM me if you need some easy to read DIY books written by attorneys about filing suit in superior court and DIY litigation in the USA

None of this is fact or legal advice. Merely the colorful, exaggerated opinion of one person for purposes of parody.

**Comment by chermany4ever** [comment]
Beyond what happened with lard ass, Feodor does not want to carry on with mypal. He said it very clearly and more than once. End of the story.

**Comment by CsacsiSoft** [comment]
Köszönöm a kitartásod, a lelkesedésed és az erre a project-re szánt időd! Háttérben követtelek és sajnálom hogy bedarált a rendszer! Igyekszem nem régi megoldásokat keresni, de sajnos a rendszer nem engedi hogy többet keressek egy új gépre! Sajnos az élet engem is bedarál és nagy a szakadék, amit átugrani nem lehet! Idézet egy elhunyt művésztől "Aki ma lent van, lent is marad!" Mégegyszer köszönöm az eddigi munkád és időd! 
G. translate: Thank you for your perseverance, your enthusiasm and time for this Project! In the background I followed and regret it was embarked on the system! I try not to look for old solutions, but unfortunately the system does not allow you to look for more on a new machine! Unfortunately, life meets me and is great in the gap that you can not skip! Quote from a deceased artist "who is down today, will remain below!" Even more thank you for your work and time!

**Comment by ghost** [comment]
CsacsiSoft
Google sucks, use DeepL.

Thank you for your perseverance, your enthusiasm and your time for this project! I've been following you in the background and I'm sorry to hear that you've been messed up! I'm trying to look for non-old solutions, but unfortunately the system won't let me earn more money for a new machine! Unfortunately life is getting me down and there is a big gap that I can't jump over! Quote from a deceased artist "He who is down today, stays down!" Thanks again for all your time and effort!

Спасибо за ваше упорство, энтузиазм и время, потраченное на этот проект! Я слежу за вами в фоновом режиме и сожалею, что с вами произошла неприятность! Я пытаюсь искать нестарые решения, но, к сожалению, система не позволяет мне заработать больше денег на новую машину! К сожалению, жизнь меня подводит, и есть большая пропасть, через которую я не могу перепрыгнуть! Цитата покойного художника "Кто сегодня упал, тот останется лежать!". Еще раз спасибо за все ваше время и усилия!

**Comment by Zoyandium** [comment]
> If someone still want to continue updating of Mypal let me know. No coding skill is required but desirable.
> But you are to be very careful, sure you become their next target.

I am sure that I am not the only one to be interested in updating MyPal, but I am not a programmer so the instructions in the wiki seem confusing, but I a sure that it will all make sense once I start.  However, I am unclear about what the following means "There is andother file for mingw, look the issue #126".  Where is this issue #126?

**Comment by Feodor2** [comment]
>Where is this issue #126?

Gone, but this not a problem, if you have the will and time i shall explain all stuff careful.

**Comment by jbmt** [comment]
As a long-time PM and MyPal user, Though not using my XP machine anymore, I'm here to say Thank You to Feodor2, I've watched the whole story for years, Don't lose courage!

**Comment by Emayeah** [comment]
> If someone still want to continue updating of Mypal let me know. No coding skill is required but desirable. But you are to be very careful, sure you become their next target.

i just got hands on the source code of mypal and i am reading the wiki on how to compile it. i don't have much coding skill except creating a super basic calculator. wish me luck ❤️ 
-an italian mypal user
